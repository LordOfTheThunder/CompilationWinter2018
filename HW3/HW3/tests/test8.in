void TestUndefLegal(bool a) {
	a = true;
}
void TestUndefIllegal(bool a) {
	t = true;
}




void main() { print("I hate this course"); }



