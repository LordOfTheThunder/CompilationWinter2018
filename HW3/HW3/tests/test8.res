---end scope---
a BOOL -1
line 5: variable b is not defined
