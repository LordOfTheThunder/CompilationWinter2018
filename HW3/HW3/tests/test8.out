---end scope---
a BOOL -1
line 5: variable t is not defined
